### PR TITLE
Added BaseActivity class and adjusted other classes to inherit

### PR DESCRIPTION
**Pull request by dlowrey**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (documentation, refactoring, basically not a feature or a bug fix)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by dlowrey** [comment]
@asclines I added the class I forgot to add last time. Can you review/merge?

File: app/src/main/java/org/sdc/ttu/sdc_test_game_app/BaseActivity.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package org.sdc.ttu.sdc_test_game_app;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.view.Menu;
+import android.view.MenuItem;
+
+/**
+ * Created by Dane Lowrey on 3/6/2017.
```
Comment (asclines) [inline app/src/main/java/org/sdc/ttu/sdc_test_game_app/BaseActivity.java:10]: remove this

**Review by asclines — CHANGES_REQUESTED** [review]
For future reference, you don't need the whole body of the PR template. Just the parts you actually fill out

File: app/src/main/java/org/sdc/ttu/sdc_test_game_app/BaseActivity.java (hunk 1 above)
Comment (dlowrey) [reply app/src/main/java/org/sdc/ttu/sdc_test_game_app/BaseActivity.java:10]: I could have sworn I changed those settings. Ugh.

**Review by asclines — APPROVED** [review]

**Pull request merged**